### PR TITLE
New head of security weapon:Energy Shotgun

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

I added an experimental energy shotgun with three fire modes:
Wide fire(lethal)
Narrow fire(lethal)
Disabler(non lethal)

Original PR is:
https://github.com/space-wizards/space-station-14/pull/24491

I closed that one because the branch was fucked up, had too many issues with it, commit history is there
## Why / Balance

Was speaking with flare about this, the HoS really needed a unique weapon instead of a bland SMG that can be obtained in other ways. This will hopefully cause more playstyle variety for HoS. The weapon is also high risk, and a steal objective for syndies. My original idea was to add the X-01 from SS13, but flare showed me this shotgun and I instantly loved it. 

The energy shotgun has seven shots, as well as a recharge taking 6 seconds per shots. I ran some tests with the energy shotgun in comparison to the WT550. The energy shotgun does not fire that fast, and while the most damaging mode does 50 heat damage, that is from close range. Alternatively, the WT550, from close range, firing a burst of five, does a similar amount of damage, while being able to do that attack six times, and being easily reloaded. Still, this does not make the energy shotgun weak, as it does well in two categories:
-Armor
-Moths and Dionae

Starting with Armor. The standard armor vest, as well as the web vest, are great against the three brute damage types, with a webvest being especially good against bullets. This weapon, however, does heat damage. This makes it stronger in those scenarios. Naturally, there are also moments where it's weaker. For example, when fighting an elite suit, or against someone with a heat shield. 

The energy shotgun also excels against moths and dionae, due to their heat weaknesses. 

Also, the main weakness of the energy shotgun will most likely be reflects. It's an energy based weapon, meaning it gets deflected by things such as eswords and eshields. I will also be balancing the weapon after it releases based on feedback.
## Media

https://github.com/user-attachments/assets/6b513979-be16-4a79-a25d-111223132978

image

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: The HoS has been given a new experimental weapon:An energy shotgun with multiple fire modes. This experimental tech is highly wanted by the syndicate, and agents will try to steal it.
- remove: Removed the Head of Security's Emergency Orders.

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
Put the emergency code change into a separate PR.

**Comment by Hanzdegloker** [comment]
Love the idea and versatility aspect but I am unsure if it will be very useful against nukies vs just grabbin like a rifle or shotgun due to the lack of reloading capabilities for prolonged encounters. Against traitors and general HoS duties seems fine and fit though. Hell zombies better watch out.

**Comment by Boaz1111** [comment]
> Put the emergency code change into a separate PR.

Emo explicitly told me to put it in the same PR.
image

**Comment by Boaz1111** [comment]
> Love the idea and versatility aspect but I am unsure if it will be very useful against nukies vs just grabbin like a rifle or shotgun due to the lack of reloading capabilities for prolonged encounters. Against traitors and general HoS duties seems fine and fit though. Hell zombies better watch out.

We'll see as time goes on I suppose.

**Comment by Brandon-Huu** [comment]
> > Put the emergency code change into a separate PR.
> 
> Emo explicitly told me to put it in the same PR. image

Emo hasn't read the pull request guidelines? 🤨

**Comment by PrivateJoe09** [comment]
> Well for Traitors, the shotgun and majority of HoS's weapons will get disabled with one EMP use.... that leaves him with his knife and mk, which in most cases will not be enough to kill someone.

Adding that if it has to be wielded means no riot shields, which is a significant drawback. One of the strengths of the WT is its reliability, unlike an EMPable shotgun. I would really like to keep the WT; otherwise, it’s hard to see this as a security buff. It's similar to how the HoS from TG has both the X-01 and the Compact Shotgun.

**Comment by Brandon-Huu** [comment]
> > Well for Traitors, the shotgun and majority of HoS's weapons will get disabled with one EMP use.... that leaves him with his knife and mk, which in most cases will not be enough to kill someone.
> 
> Adding that if it has to be wielded means no riot shields, which is a significant drawback. One of the strengths of the WT is its reliability, unlike an EMPable shotgun. I would really like to keep the WT; otherwise, it’s hard to see this as a security buff. It's similar to how the HoS from TG has both the X-01 and the Compact Shotgun.

Where in the PR does it say the WT was removed?

**Comment by Boaz1111** [comment]
> > > Well for Traitors, the shotgun and majority of HoS's weapons will get disabled with one EMP use.... that leaves him with his knife and mk, which in most cases will not be enough to kill someone.
> > 
> > 
> > Adding that if it has to be wielded means no riot shields, which is a significant drawback. One of the strengths of the WT is its reliability, unlike an EMPable shotgun. I would really like to keep the WT; otherwise, it’s hard to see this as a security buff. It's similar to how the HoS from TG has both the X-01 and the Compact Shotgun.
> 
> Where in the PR does it say the WT was removed?

it will be from his office

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -713,3 +713,47 @@
   - type: Appearance
   - type: StaticPrice
     price: 750
+
+- type: entity
+  name: energy shotgun
+  parent: [BaseWeaponBattery, BaseGunWieldable]
+  id: WeaponEnergyShotgun
+  description: An one of a kind prototype energy weapon using various shotgun configurations. The possibility for both lethal, and non-lethal shots makes this a versatile weapon.
```
Comment (NakataRin) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:721]: Some flavor text changes I'd like to offer:

I think that looks better

```suggestion
  description: A one-of-a-kind prototype energy weapon that uses various shotgun configurations. It offers the possibility of both lethal and non-lethal shots, making it a versatile weapon.
```
This version is closer to yours, just fixes some typos here and there
```suggestion
  description: A one-of-a-kind prototype energy weapon that uses various shotgun configurations. The possibility of both lethal and non-lethal shots makes it a versatile weapon.
```

**Comment by Boaz1111** [comment]
appreciated.

**Comment by Cojoke-dot** [comment]
> Well for Traitors, the shotgun and majority of HoS's weapons will get disabled with one EMP use.... that leaves him with his knife and mk, which in most cases will not be enough to kill someone.

I mean... that's what happens with every other secoff right? All you gotta do is dodge the emp, ez

**Comment by Reese1243** [comment]
> > Well for Traitors, the shotgun and majority of HoS's weapons will get disabled with one EMP use.... that leaves him with his knife and mk, which in most cases will not be enough to kill someone.
> 
> I mean... that's what happens with every other secoff right? All you gotta do is dodge the emp, ez

I was gonna mention this. it is the same situation if either a secoff or HoS gets EMP'd. both lose their stuns and both would have to rely on the pistol, knife, or seclite (depending on the situation of course but this is assuming they didnt get stuff from the armory)

**Comment by Cojoke-dot** [comment]
Uh... It's 5 shots in the shotgun till you need to reload right? It takes 7 shots consecutively of the disabler smg bolts to take someone out, so uh... 3 hits with with 2-3 pellets on each shot. I don't see much reason to use this over a normal disabler or a stun baton.
- baton, 20 charged, 3 to stun (~ 6.66 stuns), melee range
- disabler, 10 charges, 4 to stun (~ 2.5 stuns)
- disabler smg, 25 charges, 7 to stun (~3.5 stuns)
- shotgun, 5 charges (shared with your lethal mode), 3 to stun ( 1.9 stuns with all triple hits)

Maybe 20 stam per non-lethals shot? Instead of a 2-2-3, you can hit a 2-3 or a 2-2-1? It should be better than other sec gear options correct? It knocks the shotgun up to a 2.6 stun possible and gives it much more flexibility. It gives it some kind of advantage over the other options HoS has for disabling people.

Uh... for the lethal maybe put one more bullet in the wide shot? Hitting all your shots from a regular lethal round deal 60 🤷‍♂️

Finally... GUAH!!! YOU CAN'T PUT IT IN YOUR BACK SLOT!!! 😞

**Comment by Boaz1111** [comment]
> Finally... GUAH!!! YOU CAN'T PUT IT IN YOUR BACK SLOT!!! 😞 

You literally can though? Have you tested it?

**Comment by Cojoke-dot** [comment]
> > Finally... GUAH!!! YOU CAN'T PUT IT IN YOUR BACK SLOT!!! 😞
> 
> You literally can though? Have you tested it?

oh... lol, I had the disabler yml pulled up on my screen when I was writing that, got a little mixed up

**Comment by Doc-Michael** [comment]
Is it just me or do the five shots seem very low, especially for a Special head item, Krammer has 4, Enforcer has 7 and they are easily reloadable, wouldn't 8 or 10 that it originally had be better? missing your shots will be heavly punishing due to the low capacity, where fighting two people requires you to hit all your shots with majority if not all the bolts. In nukies like a previous comment stated its not a practical gun,  and even in traitors im starting to question its practical use being underepowered where a krammer can output more damage than the Energy Shotgun, and if its not practical to use, what stops a HoS from leaving it in armoury in favour of other weapons in the safes?

**Comment by PrivateJoe09** [comment]
> Is it just me or do the five shots seem very low, especially for a Special head item, Krammer has 4, Enforcer has 7 and they are easily reloadable, wouldn't 8 or 10 that it originally had be better? missing your shots will be heavly punishing due to the low capacity, where fighting two people requires you to hit all your shots with majority if not all the bolts. In nukies like a previous comment stated its not a practical gun, and even in traitors im starting to question its practical use being underepowered where a krammer can output more damage than the Energy Shotgun, and if its not practical to use, what stops a HoS from leaving it in armoury in favour of other weapons in the safes?

or just make it reload itself

**Comment by Boaz1111** [comment]
> Is it just me or do the five shots seem very low, especially for a Special head item, Krammer has 4, Enforcer has 7 and they are easily reloadable, wouldn't 8 or 10 that it originally had be better? missing your shots will be heavly punishing due to the low capacity, where fighting two people requires you to hit all your shots with majority if not all the bolts. In nukies like a previous comment stated its not a practical gun,  and even in traitors im starting to question its practical use being underepowered where a krammer can output more damage than the Energy Shotgun, and if its not practical to use, what stops a HoS from leaving it in armoury in favour of other weapons in the safes?

In traitors it's certainly not underpowered due to the prevalence of webvests

**Comment by Boaz1111** [comment]
Yeah I'll fix that

**Comment by keronshb** [comment]
I still feel like this should go to the Warden and then either the Multiphase or the Big Iron should go to the HoS.

**Comment by Pumkin69** [comment]
i think it should auto recharge like caps laser pistol but be 2x5 so you have to have a duffel or backpack or carry it on your back. The current implementation is a hard nerf to HoS if you remove the wt550. However making it recharge would be good.

**Comment by EdenTheLiznerd** [comment]
> i think it should auto recharge like caps laser pistol but be 2x5 so you have to have a duffel or backpack or carry it on your back. The current implementation is a hard nerf to HoS if you remove the wt550. However making it recharge would be good.

I'm not exactly comfortable with making satchels users suffer for using the cooler bag, Just make it take up an entire side of the satchel.

**Comment by Boaz1111** [comment]
> Many players who play as Security and a handful of HoS mains (rare find) believe that if the shotgun were to replace the WT, it would be a significant downgrade. I strongly oppose this change. The shotgun might be a "cool" weapon, but it's not better than a mere Kammer. At most, it can deal 50 heat damage from close range, totaling a disappointing 250 damage with a full magazine. Plus, you need to return to sec to recharge it, which is a hassle.
> 
> For comparison, the "bland SMG" can deal up to 480 piercing damage per magazine. You get three magazines, and it takes around 1.2 seconds to crit an unarmored human with only seven shots, leaving them bleeding as well.
> 
> Heat damage is also a joke. Incendiary bullets for the WT deals 14 heat and 2 blunt damage. Claiming the shotgun is good because it can deal with webvests limits its usefulness to "it can kill webvests, so it's good enough." However, it can't even crit an elite hardsuit, even with perfect hits from all five shots in a full magazine. You'll likely get EMPed before you can do that, making it situational. The WT can still crit a webvest with 21 shots, even if the web is their counter.
> 
> The only advantage the shotgun has over the WT is that it's a simple disabler. However, the disabler SMG already fulfills this role for traitors rounds. Additionally, no sane tot player would choose getting a webvest if they were tasked with obtaining the shotgun. Antagonists already have superior close-range weapons, like the hypopen, which can instantly send you to sleep, or the esword, which you are not winning against with a shotgun.
> 
> To sum it all up, I don't think this weapon is anywhere near capable of replacing the WT or being useful for a competent HoS player, even as a stun gun. Who would choose using a shot from a 5-ammo shotgun over a disabler? It’s outmatched by most guns in the armory. Either make it stronger or beg the mappers to keep the WT.

made it a 2x4, gave it 7 shots, gave it a recharge

**Comment by PrivateJoe09** [comment]
> made it a 2x4, gave it 7 shots, gave it a recharge

this is much better, thanks.

**Review by keronshb — CHANGES_REQUESTED** [review]
> I still feel like this should go to the Warden and then either the Multiphase or the Big Iron should go to the HoS.

Reiterating this. I'd still feel way more comfortable giving this to the Warden and then introducing the Big Iron & a nerfed Revolver for the HoS.

Big Iron is also known as the MWS-01 - https://github.com/Citadel-Station-13/Citadel-Station-13/pull/14353

**Comment by Boaz1111** [comment]
Why's that?

**Comment by keronshb** [comment]
> Why's that?

The Warden gets easier home defense with an Energy Shotgun or for riot control. Especially if they also get a ballistic shotgun.

MWS-01 is a better version of the Multiphase. Higher skill ceiling and more rewarding. Also easier transport due to item size.

**Comment by Doc-Michael** [comment]
> > Why's that?
> 
> The Warden gets easier home defense with an Energy Shotgun or for riot control. Especially if they also get a ballistic shotgun.
> 
> MWS-01 is a better version of the Multiphase. Higher skill ceiling and more rewarding. Also easier transport due to item size.

I'd say that the Energy Shotgun is a much better option of us at its current standing, having original content is much better than copying over what other people already have, its a nice new concept which has potential, and its being balanced well enough where its not going to be over/underpowered, its larger size blocks it from being stored in a storage implanter thus providing a bigger challenge for any Syndicate agent trying to steal it.

Warden having a special items is probably outside of this PR's talking point and shouldn't be mentioned, there was a discussion on the Official Discord about which special HoS gun people preferred, and not once anyone mentioned the MWS-01, and don't get me wrong but the Big Iron seems like a balancing nightmare, as well as a coding one.

**Comment by keronshb** [comment]
> I'd say that the Energy Shotgun is a much better option of us at its current standing, having original content is much better than copying over what other people already have,

This is irrelevant and it's nothing new. It exists in SS13 as either technology shells to put into regular shotguns or the Warden exclusive energy shotgun.

**Comment by Boaz1111** [comment]
Right, Keron, one more question, as far as like... balancing and stuff is concerned, what is the issue with giving this to the HoS? Leaving the Warden out of the scenario for now. Like, anything I could change?

**Comment by keronshb** [comment]
> Right, Keron, one more question, as far as like... balancing and stuff is concerned, what is the issue with giving this to the HoS? Leaving the Warden out of the scenario for now. Like, anything I could change?

It's multiphase but shotgun. Having a regular shotgun that can swap between some lasers and disablers is extremely good to have in one package. These are better off to be delegated to technology shells, which it would require a different ammo type to fire these kind of projectiles.

In contrast, the Particle Defender (laser and disabler energy shotgun) was a "pump and use" weapon with a recharge rate.

**Comment by Boaz1111** [comment]
well, but like, what makes it:
Good for ward to have
Bad for HoS to have

**Comment by Boaz1111** [comment]
is it too good for the HoS?

**Comment by Pumkin69** [comment]
I'd rather the hos have the energy shotgun. Also this replaces the emergency orders which do literally nothing and give the HoS and actually interesting head item. I highly disagree with the sentiment of warden getting this instead.

**Comment by Boaz1111** [comment]
ALT DESIGN:
Cell Driven Energy Shotgun
Instead of an internal battery, this design proposes special power cells for the energy shotgun.
These power cells would hold 3-5 shots
Pros:
-More playstyle variance.
If the Head of Security wanted to focus on disabling hostiles instead of killing them, they could take more disabler cells.
-Certain about what mode is selected.
You know what cell you put in. Could also be adressed with visualisers.
-Higher skill ceiling.
This change would allow for more tactical play. See point 1.
-New unlocks for research.
In order to make the shotgun as customisable as a normal shotgun, it could be able to unlock more modes as sci researches things. Think a radiation mode, an incendiary mode, etc. 

Cons:
-Difficult to balance.
The cells are proposed to be 2x2, idea might change. If a head of security can carry many cells, they can be extremely versatile. Needs to be looked at if we go with this idea
-Harder to use. 
While higher skill ceiling might be an upside, this weapon could also be frustrating for newish heads of security. 

Overall I think it's a good idea, but needs maintainer discussion.

**Comment by Boaz1111** [comment]
To add on, the normal types would be converted into cells, the head of security would spawn with 2-3 of each, and they will be available to be made at the secfab.

**Comment by Emisse** [comment]
seems fine to me fix goncflicts

**Comment by Boaz1111** [comment]
got it

**Comment by Boaz1111** [comment]
need to unscrungle this shit

**Pull request merged**